### PR TITLE
GTEST: Fix compilation.

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/9880/ for details (Mellanox internal link).

**Comment by alinask** [comment]
@yosefe Can you please review?

File: test/gtest/Makefile.am [hunk 1]
```diff
@@ -160,16 +160,21 @@ gtest_SOURCES = \
...
 gtest_LDADD += \
 	$(IBVERBS_LDFLAGS) \
 	$(top_builddir)/src/uct/ib/libuct_ib.la
+if HAVE_DEVX
 gtest_SOURCES += \
 	uct/ib/test_devx.cc
+endif
+if HAVE_MLX5_HW
+gtest_SOURCES += \
+    uct/ib/test_ib_md.cc
```
Comment (yosefe) [inline test/gtest/Makefile.am]: minor: indentation should use tab instead of 4 spaces

Comment (alinask) [reply test/gtest/Makefile.am]: :+1:  I'll fix this in my next PR.

**Pull request merged**